src/components/TutorialMetadata.tsx

```diff
@@ -104,70 +34,126 @@ const TutorialMetadata: React.FC<IProps> = ({ tutorial }) => {
   const address = frontmatter.address
 
   return (
-    <Container>
-      <TagsContainer>
-        <PillContainer>
+    <Flex
+      flexDirection="column"
+      justifyContent="space-between"
+      borderBottomWidth={{ base: 0, lg: "1px" }}
+      borderBottomStyle={{ lg: "solid" }}
+      borderBottomColor={{ lg: "border" }}
+    >
+      <Flex justifyContent="space-between" alignItems="center" w="full" mb={8}>
+        <Flex flexWrap="wrap" w="full">
           <TutorialTags tags={frontmatter.tags} />
-        </PillContainer>
-        <SkillPill isSecondary={true} className="" color="">
+        </Flex>
+        <Flex
+          as={Pill}
+          alignSelf="flex-start"
+          mb={2}
+          whiteSpace="nowrap"
+          isSecondary={true}
+          className=""
+          color=""
+        >
           <Translation id={getSkillTranslationId(frontmatter.skill)} />
-        </SkillPill>
-      </TagsContainer>
-      <HorizontalContainer>
+        </Flex>
+      </Flex>
+      <Flex
+        mb={6}
+        flexWrap="wrap"
+        mt={-4}
+        fontSize="sm"
+        color="text300"
+        justifyContent="flex-start"
+      >
         {author && (
-          <DataContainer>
-            <Emoji size={1} mr={`0.5em`} text=":writing_hand:" />
+          <Box mr={4}>
+            <Emoji fontSize="sm" mr={2} text=":writing_hand:" />
             {author}
-          </DataContainer>
+          </Box>
         )}
         {hasSource && (
-          <DataContainer>
-            <Emoji size={1} mr={`0.5em`} text=":books:" />
+          <Box mr={4}>
+            <Emoji fontSize="sm" mr={2} text=":books:" />
```

Comment:
Hey @TylerAPfledderer thanks a lot for that :rocket: !
I'm not sure to understand what I should do about it haha :smile:

Reply:
I think the margin should be reverted back to `mr={"0.5em"}`, unless @pettinarip might find this to be a non-issue either way. This is simply a nitpick from me! 😅

Reply:
Let's see with the boss :wink: :joy: 
But in this case we should change [the migration documentation](https://github.com/ethereum/ethereum-org-website/blob/dev/docs/chakra-migration-guide.md) about emojis I guess

Reply:
I don't have a strong opinion & probably you are right @TylerAPfledderer thanks for noting that. But since the difference here is minimal I would prefer to use the Chakra values to be consistent with the rest of the migration.